Stylesheets/utility.css

```diff
@@ -0,0 +1,38 @@
+:root {
+    --flex-gap : 1rem;
+
+    /* border radius */
+    --container-border: 0.3rem;
+
+
+    /* transitions */
+    --hover : 0.5s
+}
+
+body {
+    height:100vh;
+    margin: 0 auto;
+}
+
+div{
+    padding: 1rem;
+    margin: 2rem;
+}
```

Comment:
I don't think soo it's a good practice to give by default padding and margin to all the divs.
Correct me if I'm wrong.

Reply:
Actually, I am not sure if it's a good practice but I used it considering that I will be needing this in most of my code. Thank you for pointing this out, I'll read about it and do the rectifications.